tests/tnimib.nim

```diff
@@ -85,3 +85,16 @@ suite "nbClearOutput":
     check nb.blk.context["output"].vString == ""
 
 
+nbInitPython()
```

Comment:
let's put also this behind a `whenIsAvailable(nimpy)` switch (should we have a `whenIsAvailable` template/macro somewhere?). I want to make sure that to run tests you only need the required dependencies (then CI will double check)

Reply:
Good idea 👍 I can't seem to find any good place to put small internal templates like this so it's in the test file for now unless you have a better location

Reply:
let's put it directly in `src/nimib.nim` (and use it also to hide - instead of giving error - both nbInitPython and nbPython). I plan to use it also elsewhere

Reply:
Done ✔️